### ISSUE TITLE
Add a Tutorials.md file

### ISSUE THREAD

**Issue by aspittel** [aspittel/learn-code-from-us#55]
[Move the React tutorials to their own file](https://github.com/aspittel/learn-code-from-us/issues/26#issuecomment-431651339). I really like @GaProgMan's idea:

```
I wonder whether there should be a separate markdown file for tutorials. Perhaps something like:

# Tutorials

The following tutorials are separated by technology and might be useful for contributors who are new to the technology stacks which this project uses, and would like to learn more before contributing.

### React

#### Beginners/Intro to React

#### More advanced/Specific React Topics

### HTML & CSS

#### Similar Headers Here

### A.N.Other technology

```

**Comment by aspittel** [comment]
I think we are good to move to this now!

**Comment by Rohitkrops** [comment]
I am going to work on this.

**Comment by Ifyokoh** [comment]
Can I work on this?

**Comment by aspittel** [comment]
@Ifyokoh go for it!

**Comment by Ifyokoh** [comment]
Hi @aspittel
Just made a pull request to this issue, review #60

**Comment by GaProgMan** [comment]
Fixed by #60
